Revisions to address reviewer comments (#128)
greenelab/iscb-diversity-manuscript: content/25.discussion.md

## Discussion

There are significant technical and ethical challenges that one faces in carrying out retrospective work to examine the fairness of scientific practices.

> A major technical challenge was to narrowing down geographic origins for some groups of names.

Reviewer: ```suggestion
A major technical challenge was to narrow down geographic origins for some groups of names.
```

> Some name origin groups, such as Hispanic names, are geographically disparate.

Reviewer: ```suggestion
Specific name origin groups, such as Hispanic names, are geographically disparate.
```

We were unable to construct a classifier that could distinguish names from Iberian countries (Spain and Portugal) from those in Latin America in the group of Hispanic names.
Discrepancies in representation between these groups are thus undetectable by our classifier.
Honoree counts of those with Hispanic names are influenced from Spain as well as Latin America.
In such cases, our analyses may substantially understate the extent to which minoritized scientists are underrepresented among honorees and authors.
Another technical challenge is that supervised machine learning approaches are neither error free nor bias free.

> By integrating different lines of evidence and preserving uncertainty by analyzing prediction probabilities rather than applying a hard assignment for each prediction, we aimed to alleviate method-specific biases and discover insightful findings that correctly reflect the current representation diversity at conferences.

Reviewer: 💯

A key ethical challenge with retrospective work to examine disparities is that algorithmic approaches to infer characteristics are often required.
This leads to significant limitations such as considering gender as a binary variable.
While this situation limits retroactive examination, there is a substantial opportunity for scientific societies, grant-making organizations, publishers, and others to proactively collect self-identified demographic information.

> A key element is then implementing processes to use these plans to continuously evaluate practices.

Reviewer: I'm trying to find a way to say this right and avoid repetitions. Here's a suggestion but can be adjusted.

```suggestion
It is equally crucial that this information is properly used to continuously evaluate inclusion practices.
```

> A challenge that straddles the ethical and technical divide for studies examining the representation of honorees is that the background that can be best assessed, which we assess here, is the current field composition.

Reviewer: We used "challenge" to open the last three paragraphs. How about "difficulty"?

Author: Yes - would accept a suggestion but will also try to remember to come back to this line :)

> A challenge that straddles the ethical and technical divide for studies examining the representation of honorees is that the background that can be best assessed, which we assess here, is the current field composition.

Author: ```suggestion
A difficulty that straddles the ethical and technical divide for studies examining the representation of honorees is that the background that can be best assessed, which we assess here, is the current field composition.
```

Scientific societies exist to promote the discipline, and many, including ISCB, include diversity as a value.

> In these cases the ideal background distribution would be the portion of the population that would be senior computational biologists in the absence of systemic barriers to participation.

Reviewer: ```suggestion
In these cases, the ideal background distribution would be the population of senior computational biologists in the absence of systemic barriers to participation.
```

The implication would be that scientific societies exist to reflect what the field could be, not just what it is.
However, we are limited to measuring the field as it is.
Furthermore, authorships, which we use to assess the field's composition, are also affected by systemic barriers to participation.

> We estimated the composition of the field using last author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956] women are underrepresented in this position.

Reviewer: ```suggestion
We estimated the composition of the field using last author status, but in neuroscience [@doi:10.1101/275362] and other disciplines [@doi:10.1371/journal.pbio.2004956], women are underrepresented in this position.
```

Meanwhile, a review of women's underrepresentation in math-intensive fields argued that today's underrepresentation is not explained by historic forms of discrimination but factors surrounding fertility decisions and lifestyle choices, whether freely made or constrained by biology and society [@doi:10.1073/pnas.1014871108].
A recent analysis of gender inequality across different disciplines showed that, although both gender groups have equivalent annual productivity, women scientists have higher dropout rates throughout their scientific careers [@doi:10.1073/pnas.1914221117].
Therefore, although we found that ISCB's honorees and keynote speakers appear to have similar gender proportion to the field as a whole, the gender proportions have not reached parity.

> To the extent that this is due to systemic barriers, the process may have reached equality but not equity.

Reviewer: ```suggestion
To the extent that this gap is due to systemic barriers, the process may have reached equality but not equity.
```
